README.md

```diff
@@ -1,8 +1,6 @@
 # ![Zaczero/2Captcha logo](https://github.com/Zaczero/2Captcha/blob/master/resources/2captcha.png)
 
-![github version](https://img.shields.io/github/release/Zaczero/2Captcha.svg)
-![nuget version](https://img.shields.io/nuget/v/2CaptchaAPI.svg)
-![license type](https://img.shields.io/github/license/Zaczero/2Captcha.svg)
+[![github version](https://img.shields.io/github/release/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/releases/latest) [![nuget version](https://img.shields.io/nuget/v/2CaptchaAPI.svg)](https://www.nuget.org/packages/2CaptchaAPI/) [![license type](https://img.shields.io/github/license/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/blob/master/LICENSE)
```

Comment:
Is there a reason why everything is in one line? It's hard to read long 1-line code.

```suggestion
[![github version](https://img.shields.io/github/release/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/releases/latest)
[![nuget version](https://img.shields.io/nuget/v/2CaptchaAPI.svg)](https://www.nuget.org/packages/2CaptchaAPI/)
[![license type](https://img.shields.io/github/license/Zaczero/2Captcha.svg)](https://github.com/Zaczero/2Captcha/blob/master/LICENSE)
```

Reply:
I was using https://stackedit.io/ as a markdown editor. The preview showed the badges on different lines as well.